tests/tracing_test.go

```diff
@@ -238,3 +378,15 @@ func (m *mockTracer) verifySpans(spanNames ...string) error {
 
 	return nil
 }
+
+func (m *mockTracer) getOrderedSpan() []string {
```

Comment:
`cloneOrderedSpans`?

Reply:
How about just `spans` since `get` is not idiomatic, and i'm not sure the caller needs to know that it is cloned?

Reply:
The current code distinguishes between "spans" and "ordered spans," and the type already has an `orderedSpans` field. Otherwise, I'd go with `orderedSpans`. It's good for the caller to know the cost of an operation: cloning.

Reply:
Sure, amended in 8ab99ffc5a249f83adc1a44461b4908ae167f63b